BibCnrs/BibCnrs
# Fix(header): Language selector and theme toggle now persist value as expected

## Problem

The language selector in the header was not persisting the value, leading to unepxected lost of changes

## Soluton

Persist language update if user is logged in with  ajanus a janus account

## How to Test

- Test toggle being logged out to ensure that everything work for anonymous users
- Log in with a janus account and test language and theme change
